### PR TITLE
Update docs with the new changes

### PR DESCRIPTION
**Pull request by ashensw**
Resolves https://github.com/wso2/product-is/issues/11730

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -69,9 +65,8 @@ Skip this part if you are using developer/free TypingDNA account.
 
 ### Configuring the TypingDNA in WSO2 Identity Server
 
-1. Login to console.
-2. Go to `Manage -> configurations -> other settings`
-3. Select TypingDNA Configuration.
+1. Start the WSO2 Identity Server and log in to the management console using admin credentials.
+2. Go to `Identity Providers -> Resident -> Other settings -> TypingDNA Configuration`
```
Comment (nilminiwso2) [inline docs/config.md:69]:
```suggestion
2. Go to `Identity Providers -> Resident -> Other settings -> TypingDNA Configuration`.
```

File: docs/config.md [hunk 2]
```diff
@@ -69,9 +65,8 @@ Skip this part if you are using developer/free TypingDNA account.
...
 4. Enable TypingDNA & configure API Key, Secret. You can get the Key & Secret from TypingDNA
```
Comment (nilminiwso2) [inline docs/config.md:70]:
```suggestion
4. Enable TypingDNA and configure the **API key** and **secret**. You can get the key and secret from the TypingDNA [dashboard](https://www.typingdna.com/clients/).
```

File: docs/config.md [hunk 3]
```diff
@@ -69,9 +65,8 @@ Skip this part if you are using developer/free TypingDNA account.
...
    [dashboard](https://www.typingdna.com/clients/).
```
Comment (nilminiwso2) [inline docs/config.md:71]:
```suggestion
```

File: docs/config.md [hunk 4]
```diff
@@ -69,9 +65,8 @@ Skip this part if you are using developer/free TypingDNA account.
...
    Refer [this doc](files/Sign%20In.pdf) for detailed information.
```
Comment (nilminiwso2) [inline docs/config.md:72]:
```suggestion
   [Learn more](files/Sign%20In.pdf).
```

File: docs/config.md [hunk 5]
```diff
@@ -83,9 +78,41 @@ Skip this part if you are using developer/free TypingDNA account.
 
 ### Configuring the application to use TypingDNA
 
-1. Go to `Develop -> Application` & Select the sample application you have configured.
-2. Go to `Sign-in Method`.
-3. Add `Typing-Biometric-Based` script in `templates->user`.
-   Refer [this doc](files/adaptive-script-description.md) to get detailed information about TypingDNA adaptive template.
-
+1. Go to `Service Providers -> List` & Select the sample application you have configured and click `Edit`.
```
Comment (nilminiwso2) [inline docs/config.md:81]:
```suggestion
1. Go to `Service Providers -> List`, select the sample application you have configured, and click `Edit`.
```

File: docs/config.md [hunk 6]
```diff
@@ -83,9 +78,41 @@ Skip this part if you are using developer/free TypingDNA account.
...
+    });
+};
+
+// End of TypingDNA-Based.......
+```
 ![Alt_text](images/screen-shot-4.png?raw=true)
+![Alt_text](images/screen-shot-6.png?raw=true)
+![Alt_text](images/screen-shot-7.png?raw=true)
+
+ Refer [this doc](files/adaptive-script-description.md) to get detailed information about TypingDNA adaptive functions.
```
Comment (nilminiwso2) [inline docs/config.md:118]:
```suggestion
 Learn more about [TypingDNA adaptive functions](files/adaptive-script-description.md).
```

File: docs/config.md [hunk 7]
```diff
@@ -83,9 +78,41 @@ Skip this part if you are using developer/free TypingDNA account.
...
 ### Configuring the application to use TypingDNA
 
-1. Go to `Develop -> Application` & Select the sample application you have configured.
-2. Go to `Sign-in Method`.
-3. Add `Typing-Biometric-Based` script in `templates->user`.
-   Refer [this doc](files/adaptive-script-description.md) to get detailed information about TypingDNA adaptive template.
-
+1. Go to `Service Providers -> List` & Select the sample application you have configured and click `Edit`.
+2. Expand `Local and Outbound Authentication Configuration` and click `Advanced Configuration`.
+3. Configure the required authentication for two steps and use the TypingDNA adaptive script as below.
```
Comment (nilminiwso2) [inline docs/config.md:83]:
```suggestion
3. Configure two authentication steps (2FA) in the login flow and use the Typing DNA adaptive script as shown below.
```

File: docs/config.md [hunk 8]
```diff
@@ -2,20 +2,16 @@
...
-### Installing the connector & extensions  
-
-#### Adding plugins to authentication endpoint.
-
-1. Download the TypingDNA connector and other required artifacts from the [WSO2 store](https://store.wso2.com/store/assets/isconnector/list).
-3. Copy the files inside plugins folder and paste them to `<IS-Home>/repository/deployment/server/webapps/authenticationendpoint/plugins` directory.
-4. Copy the `typing-dna.js` file to `<IS-Home>/repository/deployment/server/webapps/authenticationendpoint/js` directory.
-
-#### Enable TypingDNA in the WSO2 Identity Server.
+### Enable TypingDNA in the WSO2 Identity Server
```
Comment (nilminiwso2) [inline docs/config.md:14]:
```suggestion
### Enabling TypingDNA in the WSO2 Identity Server
```

File: docs/config.md [hunk 9]
```diff
@@ -2,20 +2,16 @@
 
 To use TypingDNA with WSO2 Identity Server, first you need to configure the authenticator with WSO2 Identity Server. The following topics provide instructions on how to configure the TypingDNA with WSO2 Identity Server as a risk-based authentication (RBA) option:
 
-* [Installing the connector & extensions](#installing-the-connector--extensions)
+```
+Note: TypingDNA is supported with WSO2 Identity Server 5.12.0-alpha10 version onwards.
+```
+* [Enable TypingDNA in the WSO2 Identity Server](#enable-typingdna-in-the-wso2-identity-server)
```
Comment (nilminiwso2) [inline docs/config.md:8]:
```suggestion
* [Enabling TypingDNA in the WSO2 Identity Server](#enable-typingdna-in-the-wso2-identity-server)
```

File: docs/config.md [hunk 10]
```diff
@@ -25,7 +21,7 @@ To use TypingDNA with WSO2 Identity Server, first you need to configure the auth
     enabled_features=["security.loginVerifyData.typingDNA"]
    ```
 
-#### Deploying TypingDNA Artifacts.
+### Deploying TypingDNA Artifacts
 
 You can either download the TypingDNA artifacts or build the authenticator from the source code.
```
Comment (nilminiwso2) [inline docs/config.md:27]: - To build from the source code:
- To build the authenticator, navigate to the `identity-conditional-auth-typingdna` directory and execute the following command in a command prompt:
- Copy the `org.wso2.carbon.identity.conditional.auth.typingdna.functions-x.x.x.jar` file into the 
   `<IS-Home>/repository/components/dropins` directory and the `api#identity#typingdna#v_.war` file into the `<IS-Home>/repository/deployment/server/webapps` directory.

File: docs/config.md [hunk 11]
```diff
@@ -2,20 +2,16 @@
 
 To use TypingDNA with WSO2 Identity Server, first you need to configure the authenticator with WSO2 Identity Server. The following topics provide instructions on how to configure the TypingDNA with WSO2 Identity Server as a risk-based authentication (RBA) option:
```
Comment (nilminiwso2) [inline docs/config.md:3]: General comment: Images are not aligned with the numbering.
Screenshot 2021-11-12 at 13 14 58

**Review by nilminiwso2 — APPROVED** [review]

File: docs/config.md (hunk 11 above)
Comment (ashensw) [reply docs/config.md:3]: Addressed with https://github.com/wso2-extensions/identity-conditional-auth-typingdna/pull/15/commits/809fb935c083c4e9bed561daf6bd729580a9e074

File: docs/config.md (hunk 10 above)
Comment (ashensw) [reply docs/config.md:27]: Addressed with https://github.com/wso2-extensions/identity-conditional-auth-typingdna/pull/15/commits/809fb935c083c4e9bed561daf6bd729580a9e074

**Comment by ashensw** [comment]
Addressed the requested changes with https://github.com/wso2-extensions/identity-conditional-auth-typingdna/pull/15/commits/809fb935c083c4e9bed561daf6bd729580a9e074

**Review by omindu — APPROVED** [review]

**Pull request merged**